apps/docs/app/features/routes/ressurser/design-tokens/ShadowTokens.tsx

```diff
@@ -18,6 +32,51 @@ export function ShadowTokens(props: BoxProps) {
           Elevation 1, Elevation 2 og Elevation 3.
         </Text>
       }
-    ></SharedTokenLayout>
+    >
+      <ShadowTokensTable />
+    </SharedTokenLayout>
   );
 }
+
+type ShadowTokenTableProps = BoxProps;
+
+const ShadowTokensTable = (props: ShadowTokenTableProps) => {
+  const tokenFormatter = useTokenFormatter();
+  return (
+    <Box {...props}>
+      <Table variant="simple" colorScheme="grey">
+        <Thead>
+          <Tr>
+            <Th>Eksempel</Th>
+            <Th>Verdi</Th>
+            <Th>Kode</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {Object.entries(tokens.depth.shadow).map(([key, token]) => (
+            <Tr key={key}>
+              <Td>
+                <Box
+                  width={8}
+                  height={8}
+                  boxShadow={token.value}
+                  borderRadius="xs"
+                />
+              </Td>
+              <Td>
+                {key}/ {token.value}
```

Comment:
```suggestion
                {key} / {token.value}
```

La til et mellomrom bare jeg